docs/en/guides/pipeline-integrations/administration/setting-python-3-desktop.md

```diff
@@ -5,17 +5,17 @@ pagename: setting-python-3-desktop
 lang: en
 ---
 
-# Set Python 2 as the default Python version in {% include product %} Desktop
```

Comment:
This change is not straightforward. Python 3 is the default version since SGD 1.7.3. So we don't need a section for setting "default to py3"

Now having/keeping a section regarding running SGD in Python 2 raises some problems: SGD version 1.8 does not ship Python 2 anymore. But the docs are still valid for SGD version 1.7.3.
So we might just want to add a "version requirement" warning message saying that this is only valid for SGD version 1.7.3.

Reply:
Thanks @julien-lang ! Great idea. Fixing.

Reply:
I think you need to revert the changes of this section. The goal of this section, when using SGD 1.7.3, is to run Python 2, not 3.

Keeping the note bellow saying Py2 is no longer supported (as of version 1.8.0 of SGD).

Reply:
Thanks Julien! I'm going to clean this all up and submit another PR with all the changes.